### DISCUSSION TITLE
Report spam channel and scan comments in community posts.

### DISCUSSION THREAD

**Discussion by kiatchun** [ThioJoe/YT-Spammer-Purge#195] [Ideas]
Is there any way to report the spammers' channels while reporting the comments, and is there support for community post comments.

**Comment by Kurdakov** [comment]
no, because API does not support that ( and for some reasons google engineers refuse to support community posts in API, claiming, that feature is obsolete see https://issuetracker.google.com/u/1/issues/206519108 )

**Reply by kiatchun** [reply]
But what about reporting the spammers' channels that create those comments.
